### PR TITLE
Close span

### PR DESCRIPTION
**Pull request by skendrot**
Properly close span and remove an extra closing span

### CODE REVIEW

File: src/Generator/Generators/HtmlOmdGenerator.cs [hunk 1]
```diff
@@ -252,7 +251,7 @@ private string GetIcon(ISymbol type, string content)
                 icon += "event";
             if(type.IsStatic && type.ContainingType?.TypeKind != TypeKind.Enum)
             {
-                content = "<span class='static'/>" + content;
+                content = "<span class='static'></span>" + content;
```
Comment (dotMorten) [inline src/Generator/Generators/HtmlOmdGenerator.cs:254]: I don't get the point of this. I wasn't getting html warnings for this once the other issues were resolved.

File: src/Generator/Generators/HtmlOmdGenerator.cs [hunk 2]
```diff
@@ -214,7 +214,6 @@ public void WriteType(INamedTypeSymbol type, INamedTypeSymbol oldType)
                     if (iface.wasRemoved) sw.Write("</span>");
                     i++;
                 }
-                sw.WriteLine("</span>");
```
Comment (dotMorten) [inline src/Generator/Generators/HtmlOmdGenerator.cs:217]: Where's the opening span inside this scope?

Comment (skendrot) [reply src/Generator/Generators/HtmlOmdGenerator.cs:217]: There wasn't one. Which is why I removed this closing span

File: src/Generator/Generators/HtmlOmdGenerator.cs (hunk 1 above)
Comment (skendrot) [reply src/Generator/Generators/HtmlOmdGenerator.cs:254]: Did you have static members? A self-closing span isn't valid html

File: src/Generator/Generators/HtmlOmdGenerator.cs (hunk 2 above)
Comment (dotMorten) [reply src/Generator/Generators/HtmlOmdGenerator.cs:217]: d'oh! Brainfart. Thanks!

**Pull request merged**